googletest/include/gtest/gtest.h

```diff
@@ -2116,9 +2116,11 @@ GTEST_API_ AssertionResult DoubleLE(const char* expr1, const char* expr2,
 // of the dummy variable name, thus allowing multiple SCOPED_TRACE()s
 // to appear in the same block - as long as they are on different
 // lines.
-#define SCOPED_TRACE(message) \
-  ::testing::internal::ScopedTrace GTEST_CONCAT_TOKEN_(gtest_trace_, __LINE__)(\
-    __FILE__, __LINE__, ::testing::Message() << (message))
+#define SCOPED_TRACE_(message, file, line) \
```

Comment:
as far as I'm aware, the only way to do this would still require defining multiple macro names (eg selecting the macro impl based on number of arguments). I don't know if that's any better